### DISCUSSION TITLE
Schedule monitoring

### DISCUSSION THREAD

**Discussion by ampsu** [mcguirepr89/BirdNET-Pi#767] [Q&A] [answered]
Hello

I got two Pi-s up and running for a little while. All looks good but I was wondering if it is somehow possible to schedule the analyzer? For ex. only monitor between 7pm-9am. I really dont need midday monitoring. I could squeeze more out of my batteries if I does not run exactly 24/7.

**Comment by ehpersonal38** [comment, marked as answer]
I think the easiest way to do this will be to use cronjobs. For example, if you want the schedule to be 7pm-9am, you can add this to your crontab:
```
0 19 * * * cd ~/BirdNET-Pi/scripts/ && ./stop_core_services.sh
0 9 * * * cd ~/BirdNET-Pi/scripts/ && ./restart_services.sh
```

Here's a one-liner that'll add those above entries to your crontab:
```
{ crontab -l; echo "0 19 * * * cd ~/BirdNET-Pi/scripts/ && ./stop_core_services.sh"; echo "0 9 * * * cd ~/BirdNET-Pi/scripts/ && ./restart_services.sh"; } | crontab - && crontab -l
```

**Reply by ampsu** [reply]
Thank you! Looks like it works. Your example had stop and restart vice-versa but I figured it out :) 
It really does make a difference on a system load so I figure it helps a lot on the battery life as well.
Only thing I noticed is that it also restarts services that I got disabled (spectrogram, chart). I guess I could modify the restart script in order to keep them disabled?

**Reply by ehpersonal38** [reply]
You're welcome! I'd recommend making copies of those scripts and modifying them as needed, since any edits to the originals will be overwritten if you ever update your BirdNET-Pi.

**Comment by BastianLinge** [comment]
I want to do the same.
A Raspberrypi that analyzes birds at night and in the evening and does something else during the day when people can be heard in the gardens around us.
For a first test, it works with the cron jobs described above.
Because the was a large cpu usage, Ive added icecast2.service in the stop_core_services.sh script too.

Would it be better if the Raspberrypi is completely rebooted to start birdnetpi to ensure that all services are loaded?
